manuscript/draft.Rmd

```diff
@@ -283,83 +283,88 @@ biodiversity in the form of species richness. The results of this effort are
-across all modeling approaches. This highlights the importance evaluating models
-based on their uncertainty as well as their mean estimates. Not doing so is
-particularly problematic in this case because the summed species distribution
-models are overly confident meaning that using them would lead to believing that
-richness would be restricted to a much narrower range than would actually
-occur.
-
-The accurate estimates of uncertainty by JSDMs and richness models result in
-these models approaching the performance of time-series models and baselines at
-the longest time-scales assessed when using measures that consider both point
-estimates and uncertainty. This convergence at longer time scales highlights the
-importance of considering how forecasting method performance changes with the
-distance into the future being predicted. The value of this approach, and
-associated forecast horizons, has been raised for knowing how far into the
-future a model can be effectively used for forecasting [@petchy2015], but our
-results suggest a broader value to this approach for considering the potential
-importance of different models and processes for making forecasts at different
-scales. [[Should make sure Petchy et al. don't make this point]] Our results
-show that for forecasts 1-3 years in the future, baselines and time-series based
-approaches outperform all other methods using integrative metrics. At these time
-scales environmental changes are relatively small and changes in the biota may
-lag behind changes in the environment. However, as the timescale of the forecast
-increases to a decade the JSDMs and environmental richness models get closer to
-the accuracy of the time-series methods and even surpass some of those
-methods. As the forecast distance increases the amount of environmental change
-is expected to increase and the system will have more time to respond,
-potentially leading to shifts towards improved relative performance of the
-models incorporating environmental and ecological information. If this shift
-with forecast distance in the relative importance of different processes and
-models continues, this could lead to JSDMs and environmental richness models
-outperforming time-series based approaches at sufficiently long time
-scales. These results suggest that assessment of how forecast performance
-changes with time lag is not only important for determining how far in the
-future to forecast, but also for determining models and processes are most
-relevant for making forecasts at different time scales.
+environmental variables and using the resulting models to make forecasts []. Two
+comparisons of the the SDM and direct richness modeling approaches reported that
+the methods yielded equivalent results for forecasting diversity (some other
+comparisons that found poor SDM performance were [[tainted]] by the rounding
+error alluded to above and are not considered here). While our results generally
```

Comment:
I'll defer to you on this.  I do see a lot of rounding in the literature, which makes it harder to ignore.